# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/avview/configure.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2004-01-05 21:58:52.000000000 +0100
+++ ./configure.in	2004-08-28 23:05:28.000000000 +0200
@@ -152,7 +152,7 @@
 		#error "FFMPEG libraries mismatch"
 	#endif
 	
-	#if LIBAVCODEC_VERSION_INT != 0x000408
+	#if LIBAVCODEC_VERSION_INT < 0x000408
 		#error "AVview now requires ffmpeg 0.4.8"
 	#endif
 	
@@ -203,7 +203,7 @@
 	AC_MSG_ERROR([can not find X11])
 fi
 
-if ! test "$TCL_VERSION $TK_VERSION" = "8.4 8.4" ; then
+if ! test "${TCL_VERSION:0:3} ${TK_VERSION:0:3}" = "8.4 8.4" ; then
 	AC_MSG_ERROR([AVview requires Tcl/Tk 8.4.x])
 fi
 
--- ./configure	2004-08-28 23:30:05.000000000 +0200
+++ ./configure.patch	2004-08-28 23:31:50.000000000 +0200
@@ -7605,7 +7605,7 @@
 		#error "FFMPEG libraries mismatch"
 	#endif
 
-	#if LIBAVCODEC_VERSION_INT != 0x000408
+	#if LIBAVCODEC_VERSION_INT < 0x000408
 		#error "AVview now requires ffmpeg 0.4.8"
 	#endif
 
@@ -9555,7 +9555,7 @@
    { (exit 1); exit 1; }; }
 fi
 
-if ! test "$TCL_VERSION $TK_VERSION" = "8.4 8.4" ; then
+if ! test "${TCL_VERSION:0:3} ${TK_VERSION:0:3}" = "8.4 8.4" ; then
 	{ { echo "$as_me:$LINENO: error: AVview requires Tcl/Tk 8.4.x" >&5
 echo "$as_me: error: AVview requires Tcl/Tk 8.4.x" >&2;}
    { (exit 1); exit 1; }; }
